src/components/PostDetails.tsx

```diff
@@ -104,23 +59,23 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
       </div>
 
       <div className="block">
-        {!loaded && (
+        {loaded && (
```

Comment:
Why did you change this? Everything seems to be working fine but your code is confusing because `Loader` is shown on `loaded = true` which is illogical. Pls check the logic or consider renaming `loaded` to `loading` to make your code understandable
Do the same for all occurrence of `loaded` in this file